Misc/NEWS.d/next/Library/2023-05-03-16-51-53.gh-issue-104144.653Q0P.rst

```diff
@@ -0,0 +1 @@
+Optimize :class:`asyncio.TaskGroup` when using :func:`asyncio.eager_task_factory`. Skip scheduling done callbacks when all tasks finish without blocking.
```

Comment:
Hmm actually the scheduling is skipped on a per task basis rather than when all tasks finish without blocking.

Reply:
I have a fix in https://github.com/itamaro/cpython/commit/ac1ee82a311b86d78f422364bf8ada660df21ecb (PR depends on gh-104251 first)